### ISSUE TITLE
5.5.5 - Relative paths in SCSS are compiled with domain into CSS

### ISSUE THREAD

**Issue by blizam** [gantry/gantry5#2953]
Example:

SCSS

#g-header {
		background: url('../../../../uploads/header-backgrounds/image.png');
}

compiles to:

#g-header{background:url('https://www.domain.com../../../../uploads/header-backgrounds/image.jpng');

.... which obviously won't load correctly

Rolling back to 5.4.37 resolves the issue

**Comment by mahagr** [comment]
Are you using WP?

**Comment by blizam** [comment]
Yes.From: Matias Griese ***@***.***>Sent: Monday, November 29, 2021, 5:44 AMTo: gantry/gantry5Cc: Matt; AuthorSubject: Re: [gantry/gantry5] 5.5.5 - Relative paths in SCSS are compiled with domain into CSS (Issue #2953)
Are you using WP?

**Comment by mahagr** [comment]
Generally, I wouldn't use `..` in SCSS because the CSS files it creates can be saved into multiple locations. I would just keep all the files relative to the document root.

**Comment by blizam** [comment]
but it's not appending the domain in Gantry 5.4.37? Using relative pathing to maintain a separate dev environment and ultimately just to navigate to the main WP uploads folder so the images are accessible from the WP media manager.

I can certainly adjust the pathing in the SCSS to be non-relative but am trying to avoid it...
